src/organization.py

```diff
@@ -0,0 +1,17 @@
+from constants import IMAGE_ADDRESS
+
+
+class Organization:
+    def __init__(self, organization):
+        self.organization = organization
+
+    def serialize(self):
+        org_img = lambda x: f"{IMAGE_ADDRESS}/{self.organization['slug']}/{x}.png"
```

Comment:
nit: change to image name or smth to make more clear

Reply:
will do!